source/includes/minimum-source-privileges.rst

```diff
@@ -0,0 +1,151 @@
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :widths: 10 30
+
+   * - Sync Type
+     - Minimum Source Privileges
+
+   * - Default
+     - .. code-block:: javascript
+
+          [
```

Comment:
Here's an example of what I mean about the spacing. I hope that this is helpful! LMK if you have any questions.

https://gist.github.com/jmd-mongo/9ae39e4e46ea3d42cda606d5c803f59a